app/views/items/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_for @user.items.build, remote: true do |f| %>
```

Comment:
is this correct object to attach the form_for to? I realise it works but was a bit unsure about the syntax. Also not so clear on the distinction between @user.items.build vs @user.items.new... Maybe we can discuss this?